### ISSUE TITLE
[Suggestion]: Better Tooltip Readability

### ISSUE THREAD

**Issue by Stapleton** [ejektaflex/Bountiful#52]
### What would lead me to make this issue
Currently the tooltip looks like [this](https://cdn.discordapp.com/attachments/231527759279685634/618423655587905581/unknown.png) (Screenshot courtesy of Teeto_Kaziim#0001 in the Modded MC discord)
________
### Alleviate a hard to read glob of objectives
+1 Readability would come from making each objective its own line in the tooltip, with a bit of an indent so there would be a header like type thing going on. Example below.

**Current Tooltip Layout**
```
Required: Objective 1, Objective 2
```

**Proposed Tooltip Layout**
```
Required:
 - Objective 1
 - Objective 2
```
________
### Fold in objectives if there are a lot
As there doesnt seem to be a limit on objectives, make it so if there is more than X objectives, that it hides the objectives and are shown by holding shift. Example below.
X could be either a hardcoded max or you could put it in the config. 

**Shift is not held and X is 4**
```
Required:
 - Objective 1
 - Objective 2
 - Objective 3
 - Objective 4
 - 3 More... Hold *shift* to show
```
**Shift is held**
```
Required:
 - Objective 1
 - Objective 2
 - Objective 3
 - Objective 4
 - Objective 5
 - Objective 6
 - Objective 7
```

**Comment by ejektaflex** [comment]
I can try to do this. I'll add it to my list!

**Comment by ejektaflex** [comment]
This is completely implemented for Bountiful v3.0 for later versions of Minecraft as of d7112bf!
